classes/View.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+class View
+{
+    private $template;
+    
+    public function __construct($template, $frontback)
+    {
+        $this->template = $template;
+        $this->frontback = $frontback;
+    }
+    
+    public function render($params = array())
+    {
+        //On récupère dans le controller, le fait que c'est du backend ou du frontend
+        $frontback = $this->frontback;
+        
+        // Affichage de la topbar
+        $topbar="views/topbar_$frontback.php";
+        
+        extract($params); // extrac() parcourt le tableau de $params et créé posts, responses, post, etc.
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The use of function extract() is discouraged](https://app.codacy.com/manual/JeromeLacquemant/projet-5/pullRequest?prid=5379676)